tests/unit/sagemaker/jumpstart/estimator/test_estimator.py

```diff
@@ -447,7 +475,9 @@ def evaluate_estimator_workflow_with_kwargs(
             f"{get_training_dataset_for_model_and_version(model_id, model_version)}",
         }
 
-        expected_fit_kwargs = overwrite_dictionary({"inputs": channels, "wait": True}, fit_kwargs)
+        expected_fit_kwargs = overwrite_dictionary(
+            {"inputs": channels, "wait": True, "job_name": "none of your business"}, fit_kwargs
```

Comment:
lol, except that I don't think you can have spaces in resource names.

Does that mean the validator is not catching that the name isn't valid?

Reply:
the validator is in the base class, and we mock the call to the base class, thus why the validator is not working.

Reply:
Anyways, i'll remove the spaces so to not raise any eyebrows